libraries/api-documenter/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-documenter",
+  "version": "0.1.0",
```

Comment:
No.  For now "api-documenter" will be part of API Extractor.  When it is finished, I will add it to the .npmignore so that it gets included in the tarball for \@microsoft/api-extractor